contrib/test/integration/build/kubernetes.yml

```diff
@@ -5,6 +5,7 @@
     repo: "https://github.com/runcom/kubernetes.git"
     dest: "{{ ansible_env.GOPATH }}/src/k8s.io/kubernetes"
     version: "cri-o-patched-1.8"
+    force: "{{ force_clone | default(False) | bool}}"
```

Comment:
FWIW: This is the thing that bothers me.  Having to clone this repo. twice because caching + CI sucks.  If we always have to run slash-test ami in every PR, what's the point?